src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2699,6 +2699,11 @@
           ]
         },
 
+        "justification": {
```

Comment:
>justification [](start = 9, length = 13)

when was this proposed? i did not participate in the conversation. :) i don't want this data in the log file, personally.

Reply:
marking this by design based on chat conversation.

---
In reply to: [277075812](https://github.com/Microsoft/sarif-sdk/pull/1444#discussion_r277075812) [](ancestors = 277075812)